### DISCUSSION TITLE
Ubuntu 20.04 software repository does not link to current versions of wxmaxima!

### DISCUSSION THREAD

**Discussion by New-Pilgrim** [wxMaxima-developers/wxmaxima#1729] [General]
The current library has two versions of wxmaxima listed both pointing at out of date versions (1704 and 1907).  Furthermore, the 1704 link no longer works at all, and it seems that the 1907 version is not working correctly.  It appears that there is something wrong in the generation of the output image so that the output appears in a sort of skeleton outline form with all the variable names and alpha characters cut out. 

I have been running wxmaxima under both Windows and Ubuntu Linux for over 10  years now.  I am a retired professional engineer and find that wxmaxima is an extraordinarily powerful program.  However, while there is copious documentation to be found on line this tends to be very variable in quality and unlocking some of the full power of the program often involves much trial and error before a good approach is found.  So far loading wxmaxima has involved
loads from DVD, Ubuntu repositories, PPIs, snaps, and lately Apimages.

LibreOffice, Wikipedia, Thunderbird, and Firefox are all well managed open source projects to which I periodically make small voluntary cash contributions.  However, wxmaxima appears to lack an equivalent supportive secretariat service to stabilise the offering to the user base and provide a measure of consistency.  

Can something be done to improve access to the program for Ubuntu users, and more generally to improve the quality of support to all classes of user?

**Comment by gunterkoenigsmann** [comment]
The idea behind Ubuntu (and actually all debian flavours) is to never surprise the user by any update that extends to more than a 10-line bugfix.

wxMaxima actually is developed nearly exclusively on Ubuntu so I was confident that we don't have any severe bugs on that system. My guess is that wxWidgets and your video driver somehow disagree on how to do something. Can you post me a screenshot of the problem? As long as I have no idea on how to reproduce it it typically is hard to find out what is happening; a complete all-of-wxmaxima-in-one-file edition including half of the video driver can be found here: https://github.com/wxMaxima-developers/docker-wxmaxima/releases . Just download the file and set its "executable" bit. With a bit of luck that package includes a working copy of whatever is broken on your system...
